build.gradle

```diff
@@ -36,6 +36,10 @@ allprojects {
     ruleSetFiles = files("$rootProject.projectDir/config/pmd/ruleset.xml")
   }
 
+  jacoco {
+    toolVersion = "0.8.1"
```

Comment:
this should be the default on new gradle

Reply:
that's what i've missed - gradle upgrade